packages/radio-box-group/src/RadioBox.tsx

```diff
@@ -232,7 +232,7 @@ export default function RadioBox({
           className={interactionContainerStyle}
           disabled={disabled}
           focusTargetElement={inputElement}
-          borderRadius="1px"
+          borderRadius="2px"
```

Comment:
This probably needs to be 3px. It's not currently matching up on the corners:

<img width="559" alt="Screen Shot 2021-03-16 at 10 10 18 AM" src="https://user-images.githubusercontent.com/25213299/111323051-e9e39580-863f-11eb-97aa-745ada5197c4.png">

Reply:
updated!